index.html

```diff
@@ -1,16 +1,126 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
 
-<body>
+  <body>
     <!-- Add your markup here -->
-</body>
-
-</html>
\ No newline at end of file
+    <header>
+      <div class="header-content">
+        <div class="logo-img">
+          <img
+            src="https://www.pngitem.com/pimgs/m/535-5356719_vegan-bakery-logo-designs-hd-png-download.png"
+            alt="Cake logo"
+          />
+        </div>
+        <div class="hamburger">
+          <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </div>
+        <div class="header-text">
+          <span>The best cakes in town delivered to your door</span>
+        </div>
+      </div>
+    </header>
+    <main>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="items">
+            <a href="/index.html" class="links">Home</a>
+          </li>
+          <li class="items">
+            <a href="#" class="links">Cakes</a>
+          </li>
+          <li class="items">
+            <a href="#" class="links">Orderin</a>
+          </li>
+          <li class="items">
+            <a href="#" class="links">Lessons</a>
+          </li>
+          <li class="items">
+            <a href="#" class="links">About</a>
+          </li>
+        </ul>
+      </nav>
+      <section class="hero">
+        <h1>Welcome</h1>
+        <div class="main-text">
+          <p>
+            We are very experienced in making and decorating our vegan cakes.
+            All of our cakes are vegan, so they are all 100% free-from dairy and
+            eggs. We use a range of plant-based ingredients in our mixes and
+            icings. Also all of our cakes can be made gluten free if requested.
+          </p>
+        </div>
+        <figure class="fab-img">
+          <img
+            class="main-img"
+            src="https://thebananadiaries.com/wp-content/uploads/2020/07/homemade-vegan-wedding-cake-recipe_5088-1333x2000.jpg"
```

Comment:
This image is taller than it is wide (portrait), would be easier to match the design if you pick one that's wider than it is tall (landscape)